deployment/conf/.templates/deployment.cfg.templ

```diff
@@ -58,4 +58,16 @@ condor.submit.desc.file.path={{ default .Env.condor_submit_desc_file_path "/kb/d
 condor-submit-workdir={{ default .Env.condor_submit_workdir "/mnt/condor" }}
 
 ## Formula = Token Expiration in ms - (Time_Before_Expiration / 60 ) * 1000
-time.before.expiration = {{ default .Env.time_before_expiration "10" }}
\ No newline at end of file
+time.before.expiration = {{ default .Env.time_before_expiration "10" }}
```

Comment:
Changes not yet needed in gitlab, as these defaults are fine for now. Otherwise, we can mirror these defaults in gitlab.